Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -377,6 +391,84 @@ function Get-SpnAccessToken {
     }
 }
 
+# Get the Bearer Access Token from the Endpoint
+function Get-MsiAccessToken {
```

Comment:
Can we have single copy of this instead of duplicates for REST and commands.

Reply:
Take this comment in separate PR.